src/ui/public/courier/fetch/request/segmented.js

```diff
@@ -39,33 +39,34 @@ export function SegmentedRequestProvider(es, Private, Promise, timefilter, confi
     *********/
 
     start() {
-      super.start();
```

Comment:
This is [mostly just whitespace changes](https://github.com/elastic/kibana/pull/14017/files?w=1#diff-9a3e0be94bfa2f681e9954b995eee88c)